snapshot-cross-account-replicator/lambda_source_account.tf

```diff
@@ -0,0 +1,169 @@
+## Step 1 Lambda function:
+##  will create the initial snapshots in the source account & source region
+
+resource "aws_lambda_function" "step_1" {
+  provider         = aws.source
+  function_name    = "rds-snapshots-replicator-step-1-${var.name}"
+  role             = aws_iam_role.source_lambda.arn
+  handler          = "lambda.create_snapshots"
+  filename         = data.archive_file.lambda_zip.output_path
+  source_code_hash = data.archive_file.lambda_zip.output_base64sha256
+  runtime          = "python3.8"
```

Comment:
are the scripts compatible with python3.9?

Reply:
No idea. Is it needed?

Reply:
Not really but it doesn't harm to run on the latest version i think?

Reply:
got it working with python 3.9 and arm64 arch